frontend/simple/views/Invite.vue

```diff
@@ -1,54 +1,89 @@
+        </p>
+
         <i18n v-if="error" id="badUsername" class="help is-danger">Invalid Username</i18n>
         <i18n v-if="self" class="help is-danger">Cannot Invite Yourself</i18n>
-        <table class="table is-bordered is-striped is-narrow">
+
+        <table
+          class="table is-bordered is-striped is-narrow"
+          v-if="members.length"
+        >
           <thead>
           <tr>
             <th class="table-header"><i18n>Initial Invitees</i18n></th>
           </tr>
           </thead>
           <tbody>
-          <tr v-for="(member, index) in members" class="member">
-            <td>
-              <div class="media">
-                <div class="media-left">
-                  <p class="image is-64x64">
-                    <!-- TODO: use responsive figure:
-                  http://bulma.io/documentation/elements/image/ -->
-                    <!-- TODO: ideally these would be loaded from cache -->
-                    <img src="http://bulma.io/images/placeholders/128x128.png">
-                  </p>
-                </div>
-                <div class="media-content">
-                  <strong>{{member.name}}</strong>
-                </div>
-                <div class="media-right">
-                  <button class="delete" v-on:click="remove(index)"></button>
+            <tr v-for="(member, index) in members" class="member">
+              <td>
+                <div class="media">
+                  <div class="media-left">
+                    <p class="image is-64x64">
+                      <!-- TODO: use responsive figure:
+                    http://bulma.io/documentation/elements/image/ -->
+                      <!-- TODO: ideally these would be loaded from cache -->
+                      <img src="http://bulma.io/images/placeholders/128x128.png">
+                    </p>
+                  </div>
+                  <div class="media-content">
+                    <strong>{{member.name}}</strong>
+                  </div>
+                  <div class="media-right">
+                    <button class="delete" @click="remove(index)"></button>
+                  </div>
                 </div>
-              </div>
-            </td>
-          </tr>
+              </td>
+            </tr>
           </tbody>
         </table>
-        <div class="has-text-centered button-box">
-          <div id="errorMsg" v-if="errorMsg" class="help is-danger">{{errorMsg}}</div>
-          <div id="successMsg" v-if="invited" class="created"><i18n>Success</i18n></div>
-          <button class="button is-success is-large" v-if="!invited" :disabled="!members.length" v-on:click="submit" type="submit"><i18n>Invite Members</i18n></button>
-          <a class="button is-warning is-large" v-if="invited"><i18n>Next: ?</i18n></a>
+
+        <div class="has-text-centered">
+          <button
+            class="button is-success is-large"
+            type="submit"
+            v-if="members.length"
+          >
```

Comment:
@vijayee good spot, not sure how I missed that. Might of had to do with recent merge of `demo` that messed up all my PRs, I'll fix this now